source/patterns/storybook.css

```diff
@@ -224,7 +99,7 @@
   padding-block: var(--size-12);
 
   .font-demo {
-    @media (min-width: 768px) {
+    @media (width >= 768px) {
```

Comment:
What is wrong with using `min-width`?  What value do we get from changing the syntax?

Reply:
Range notation is what CSS standards are moving towards.  So I always try to be out in front of it.

For example: @media (100px <= width <= 1900px) is the equivalent of @media (min-width: 100px) and (max-width: 1900px) which is less verbose.